common/schema.h

```diff
@@ -487,6 +489,8 @@ namespace swss {
 
 #define STATE_FLOW_COUNTER_CAPABILITY_TABLE_NAME    "FLOW_COUNTER_CAPABILITY_TABLE"
 
+#define STATE_TX_TABLE_NAME    "TX_TABLE"
```

Comment:
better to use name that is more clear. for example: TX_ERROR

Reply:
fixed